src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -141,9 +151,23 @@ subroutine ZB2020_init(Time, G, GV, US, param_file, diag, CS, use_ZB2020)
                  "subgrid momentum parameterization of mesoscale eddies.", default=.false.)
   if (.not. use_ZB2020) return
 
+  call get_param(param_file, mdl, "USE_ANN", CS%use_ann, &
+                 "ANN inference of momentum fluxes: 0 off, 3: use ANN", default=0)
```

Comment:
What do values of 1 or 2 do? Should this be a logical (like the parameter name suggests)?